hw/ip/clkmgr/data/clkmgr.sv.tpl

```diff
@@ -49,7 +53,7 @@
   input prim_mubi_pkg::mubi4_t scanmode_i,
 
   // idle hints
-  input [${len(typed_clocks.hint_clks)-1}:0] idle_i,
+  input prim_mubi_pkg::mubi4_t [${len(typed_clocks.hint_clks)-1}:0] idle_i,
```

Comment:
Should we add a countermeasure ID label for this? E.g. `// SEC_CM: IDLE.INTERSIG.MUBI`?

Reply:
yeah we should, thanks for the reminder.

Reply:
there's also an added prim count too, forgot about that one as well.